src/github/pullRequestManager.ts

```diff
@@ -606,7 +606,8 @@ export class PullRequestManager implements IPullRequestManager {
 				comments: 0,
 				commits: 0,
 				head: data.head,
-				base: data.base
+				base: data.base,
+				mergeable: true

```

Comment:
It seems it's always `true`?

Reply:
It gets updated when the individual pull request is fetched, it isn't returned when listing all PRs. I'll change the property to optional and just not set it here instead